cloud/aws/bin/deploy.py

```diff
@@ -18,3 +18,8 @@ def run(config):
 
     aws = AwsCli(config)
     aws.wait_for_ecs_service_healthy()
+    lb_dns = aws.get_load_balancer_dns(f'{config.app_prefix}-civiform-lb')
+    base_url = config.get_base_url()
+    print(
+        f'Server is available at {lb_dns}. Ensure your CNAME record for {base_url} is correct.'
```

Comment:
**thought (non-blocking):** Is there any more info we can provide here for the person deploying on what it means for the CNAME to be correct?